google/cloud/logging_v2/handlers/structured_log.py

```diff
@@ -59,9 +60,13 @@ def format(self, record):
         Returns:
             str: A JSON string formatted for GKE fluentd.
         """
+        # remove unwanted fields
+        record = copy(record)
+        record.exc_info = None
```

Comment:
since the record object is used for logging, what is a reason to create a full copy of the record object? why not to reset the .exc_info on the argument directly?
Otherwise, consider to use a distinct variable for the copy instead of "reusing" `record`.

Reply:
I was doing some experimenting, but ended up modifying the record object directly